MantleTests/MTLTestModel.h

```diff
@@ -148,3 +148,10 @@ extern const NSInteger MTLTestModelNameMissing;
 @property (readwrite, nonatomic, assign) NSUInteger freshness;
 
 @end
+
+// This is for testing parsing MTLJSONSerializing property by default.
+@interface MTLPropertyModel : MTLModel<MTLJSONSerializing>
+@property (readonly, nonatomic, assign) NSString *property;
+@property (readonly, nonatomic, strong) MTLTestModel *mtlJSONSerializingProperty;
```

Comment:
How about calling these properties `conforming` and `nonConforming` and making their types `MTLModel<MTLJSONSerializing> *` and `MTLModel *` to make it a little more obvious what's going on?

Also, could you add empty lines between all properties?